tst/CTA.WebForms2Blazor.Tests/FileConverters/ViewFileConverterTests.cs

```diff
@@ -275,5 +267,130 @@ public void ConvertEmbeddedCode_Converts_SingExpr()
+                FileConverterSetupFixture.TestGridViewControlFilePath);
+            
+            IEnumerable<FileInformation> fileList = await fc.MigrateFileAsync();
+            FileInformation fi = fileList.Single();
+            
+            byte[] bytes = fi.FileBytes;
+            var fileContents = Encoding.UTF8.GetString(bytes);
+            
+            string newPath = Path.Combine(FileConverterSetupFixture.TestFilesDirectoryPath, "GridViewControlOnly.razor");
+            string relativePath = Path.GetRelativePath(FileConverterSetupFixture.TestProjectPath, newPath);
+            relativePath = Path.Combine("Pages", relativePath);
+
+            string expectedContents = @"<div>
+    <GridView @ref=""GridView1"" AutoGenerateColumns=""false"" ItemType=""People"" SelectMethod=""People.GetPeople"">
+        <Columns>
+            <BoundField DataField=""Name"" HeaderText=""First Name"" ItemType=""Random""></BoundField>
+            <BoundField DataField=""LastName"" HeaderText=""Last Name"" ItemType=""People""></BoundField>
+            <BoundField DataField=""Position"" HeaderText=""Person Type"" ItemType=""People""></BoundField>
+        </Columns>
+    </GridView>
+</div>
+<div>
+    <GridView AutoGenerateColumns=""false"" DataKeyNames=""CustomerID"" SelectMethod=""GetCustomers"" EmptyDataText=""No data available"" ItemType=""PleaseReplaceWithItemTypeHere"">
+        <Columns>
+            <BoundField DataField=""CustomerID"" HeaderText=""ID"" ItemType=""PleaseReplaceWithItemTypeHere""></BoundField>
+            <BoundField DataField=""CompanyName"" HeaderText=""CompanyName"" ItemType=""PleaseReplaceWithItemTypeHere""></BoundField>
+            <BoundField DataField=""FirstName"" HeaderText=""FirstName"" ItemType=""PleaseReplaceWithItemTypeHere""></BoundField>
+            <BoundField DataField=""LastName"" HeaderText=""LastName"" ItemType=""PleaseReplaceWithItemTypeHere""></BoundField>
+            <TemplateField ItemType=""PleaseReplaceWithItemTypeHere"">
+                <ItemTemplate Context=""Item"">
+                    <button type=""button"">Click Me! @(Item.Name )</button>
+                </ItemTemplate>
+            </TemplateField>
+            <ButtonField ButtonType=""Button"" DataTextField=""CompanyName"" DataTextFormatString=""{0}"" CommandName=""Customer"" ItemType=""PleaseReplaceWithItemTypeHere""></ButtonField>
+        </Columns>
+    </GridView>
+</div>";
+            
+            Assert.AreEqual(expectedContents, fileContents);
+            Assert.True(fileContents.Contains("GridView"));
+            Assert.True(fileContents.Contains(@"ref=""GridView1"""));
+            Assert.False(fileContents.Contains("asp:GridView"));
+            Assert.False(fileContents.Contains("asp:BoundField"));
+            Assert.IsTrue(fi.RelativePath.Equals(relativePath));
+        }
+
+        [Test]
+        public async Task TestViewFileConverter_DefaultAspx()
```

Comment:
This test doesn't appear to have any assertions, is it unfinished?

Reply:
Oh yeah thats just a placeholder for now, its going to be the final test to make sure every component of the viewfileconverter works properly together